packages/IMAPClient-UI.package/ICFolderDialog.class/class/sortBlocks.st

```diff
@@ -0,0 +1,8 @@
+instance creation
+sortBlocks
+	
+	^ Dictionary withAll: {
```

Comment:
Why is this whole thing on class side?

Reply:
yes it is a little bit overkill on the class side since there is no point to ensure it is a singleton